rpc/mochostagent/security/authentication/moc_mochostagent_authentication.proto

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
We can remove this file as we would not require clients accessing the hostagent

Reply:
If we remove the authentication proto, then we can't use hostctl since there will be no way to authenticate/login to the hostagent. We don't need hostctl for the agents to work, it's only for dev testing/ debugging.

Reply:
UPDATE; we will work on this after discussion with rest of the MOC team. It may have security implications.

Reply:
If it's not needed for production the lose it.